### ISSUE TITLE
Controls do not change `ControlType` accessible property value when `AccessibleRole` changes

### ISSUE THREAD

**Issue by vladimir-krestov** [dotnet/winforms#3618]
* Original [Bug: 1129920](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1129920)
* .NET Core Version: 5.0
* Have you experienced this same bug with .NET Framework?: Yes

**Problem description:**
- ControlType property for Label, GroupBox and other controls doesn't depend on the Role property. 
ControlType and Localized control type properties stay unchanged if to change AccessibleRole property value.

**Expected behavior:**
- ControlType and Localized control type properties should be changed when Accessible Role is changed.

**Minimal repro:**
- Create a .Net 5.0 app
- Add a Label to the app form
- Change its AccessibleRole value
- Run the app and check LabelAccessibleObject Localized control type accessible property using Inspect

**Comment by Amy-Li03** [comment]
Verified this issue on .NET SDK latest build: 6.0.100-alpha.1.20623.10, ControlType and Localized control type properties can change when Accessible Role is changed.
![Fixed](https://user-images.githubusercontent.com/62929087/103076526-e45e4c80-4608-11eb-8eb4-2ba602117260.png)
